### ISSUE TITLE
Using l5-repository with lumen

### ISSUE THREAD

**Issue by catalinux** [prettus/l5-repository#50]
When adding 
```php
app->register(Prettus\Repository\Providers\RepositoryServiceProvider::class);
```

I get:
 lumen.ERROR: exception 'ErrorException' with message 'Declaration of Prettus\Repository\Providers\EventServiceProvider::boot() should be compatible with Illuminate\Foundation\Support\Providers\EventServiceProvider::boot()' in /var/www/www/vendor/prettus/l5-repository/src/Prettus/Repository/Providers/EventServiceProvider.php:6

from my composer json

```json
  "require": {
    "ext-mysql": "*",
    "ext-memcached": "*",
    "laravel/lumen-framework": "5.*",
    "vlucas/phpdotenv": "~1.0",
    "prettus/l5-repository": "dev-master"
 }

**Comment by andersao** [comment]
Hi @catalinux 

Fixed, I added a specific Service Provider for Lumen . It is available in version 2.1 . *

`$app->register(Prettus\Repository\Providers\LumenRepositoryServiceProvider::class);`

**Comment by catalinux** [comment]
Good. No errors anymore. I managed to work with it without any $app->register (i am not sure what I have missed)
